js/packages/openinference-instrumentation-beeai/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@arizeai/openinference-instrumentation-beeai",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "author": "gallas.milan@gmail.com",
+  "description": "OpenInference Instrumentation for BeeAI framework",
+  "private": false,
+  "type": "module",
```

Comment:
```suggestion
```

We do not use type module in our other packages, instead we ship as commonjs by default with esm exports. We should use type module eventually, but we should be probably be consistent and make that change for all packages when we decide to do so.